integration_test/soak_test/cmd/launcher/main.go

```diff
@@ -118,6 +121,7 @@ func mainErr() error {
       type: files
       include_paths:
       - %s
+      - %s
```

Comment:
Can you include this under a different file receiver with a different name? Otherwise there will be no way to check the log throughput from the log generator with the server side log entry count metric

Reply:
done. I also checked that I did it correctly by examining the instance's logs in Pantheon.